### ISSUE TITLE
Delete periodically alerts marked as deleted

### ISSUE THREAD

**Issue by ivnsch** [Co-Epi/app-backend-rust#132]
The current solution for alerts not being restored when re-fetching in the same time interval, is to mark the rows as deleted (`deleted` column). We've to delete these rows periodically (> fetch interval).

Not urgent: this is not very important during early testing.

Open to discuss different solutions.

**Comment by duskoo** [comment]
Opinion: Exposure alerts are obsolete once the incubation period elapses (and user has no symptoms).

**Comment by ivnsch** [comment]
Yeah, we could implement a periodic task that deletes all the alerts older than 2-3 weeks, which would cover the "marked as deleted" entries. Maybe there are UX implications (users wondering why alerts disappeared)?
